src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -211,6 +231,9 @@ export class EditDataGridPanel extends GridParentComponent {
 				},
 					(error: any) => {
 						self.notificationService.error(error);
+						self.telemetryService.createErrorEvent(TelemetryKeys.TelemetryView.EditDataGrid, TelemetryKeys.TelemetryError.EditCellEndError)
+							.withAdditionalProperties({ error: error })
```

Comment:
Are you guaranteed that this error message won't contain sensitive information such as data values? If not then you shouldn't include this.